addons/mail/static/src/new/rtc/call_participant_card.xml

```diff
@@ -22,8 +22,8 @@
                         'o-isTalking': isTalking,
                         'o-isInvitation': !rtcSession,
                         }"
-                        class="h-100 rounded-circle border-5 o_object_fit_cover"
-                        t-att-src="channelMember?.avatarUrl"
+                        class="o-mail-call-participant-card-avatarImage h-100 rounded-circle border-5 o_object_fit_cover"
```

Comment:
why adding extra class? Does not seem used

Reply:
Look like bad rebase